scss/config/_scale.scss

```diff
@@ -22,83 +22,125 @@ $text-ratios: (
+/// or assigned more semantic names before applying to
+/// patterns and compoents.
+///
+/// @group config-scale
+/// @sizes
+$root-sizes: (
+  'root': 18px,
+  'responsive': calc(16px + 0.125vw),
+
+  'large': calc(%s + %s) (1rem, 0.5vw),
+  'small': 0.9rem,
+);
+
+@include config('sizes', 'root-sizes', $root-sizes);
+
+
 // Text Sizes
 // ----------
-/// A set of standard text sizes.
+/// The `reset` size (`1rem`) allows you to reset
+/// to the root font size from inside any element.
+/// We also provide a set of pattern-specific sizes
+/// to use directly in component styles.
 ///
 /// @group config-scale
 /// @sizes
 $text-sizes: (
-  'root': 18px,
-  'rel-root': calc(16px + 0.125vw),
-
-  'large-rem': 20px ('convert-units': 'rem'),
-  'base-rem': 'root' ('convert-units': 'rem'),
-  'small-rem': 16px ('convert-units': 'rem'),
-  'xsmall-rem': 'base-rem' ('times': 0.75),
-
-  'xlarge': calc(%s + %s) ('base-rem', 2vw),
-  'large': calc(%s + %s) ('base-rem', 1vw),
-  'medium': calc(%s + %s) ('base-rem', 0.5vw),
-  'base': calc(%s + %s) ('small-rem', 0.5vw),
-  'small': calc(%s + %s) ('small-rem', 0.25vw),
-  'xsmall': calc(%s + %s) ('xsmall-rem', 0.25vw),
-
-  'h1': 'xlarge',
-  'h2': 'large',
-  'h3': 'medium',
-  'quote': 'medium',
-  'pre': 'xsmall',
+  'reset': 1rem,
+
+  'h1': calc(%s + %s) (1rem, 2vw),
+  'h2': calc(%s + %s) (1rem, 1vw),
+  'h3': 'large',
+
+  'quote': 'large',
+
+  'code': 'small',
+  'footer': 'small',
+  'search': 'small',
 );
 
 @include config('sizes', 'text-sizes', $text-sizes);
 
 
 // Spacing Sizes
 // -------------
-/// A set of standard spacing sizes.
+/// A mostly-linear scale of spacing-sizes
+/// based on fractions and multiples of the base line-height.
+/// These can be used to add spacing between components,
+/// and provide site hierarchy.
 ///
 /// @group config-scale
 /// @sizes {ruler}
 $spacing-sizes: (
-  'rhythm': 'root' ('line-height': 1, 'convert-units': 'rem'),
+  'rhythm': 1rem ('line-height': 1),
   'gutter': 'rhythm',
-  'gutter-minus': 'gutter' ('times': 0.75),
-  'gutter-plus': 'gutter' ('times': 1.5),
+  'gutter-plus': 'gutter' ('plus': 'shim'),
 
   'flex-gutter': calc(%s + %s) ('shim', 2.5vw),
   'spacer': calc(%s + %s) ('gutter' ('times': 3), 2.5vw),
 
+  'gutter-minus': 'gutter' ('minus': 'shim'),
   'shim': 'gutter' ('times': 0.5),
   'half-shim': 'shim' ('times': 0.5),
-  'quarter-shim': 'shim' ('times': 0.25)
+  'quarter-shim': 'shim' ('times': 0.25),
 );
 
 @include config('sizes', 'spacing-sizes', $spacing-sizes);
 
 
 // Pattern Sizes
 // -------------
-/// A set of pattern-specific sizes,
-/// that can be applied to various components.
+/// Semantically-named sizes
+/// for managing patterns and component layouts.
 ///
 /// @group config-scale
-/// @sizes {output-only}
+/// @sizes {ruler}
 $pattern-sizes: (
-  'border': 8px,
-  'page': 50rem,
+  'nav-underline': 4px,
+  'icon': 28px,
 
-  'page-break': 50em,
-  'nav-small': 21em,
-  'nav-medium': 32em,
-  'nav-large': 65em,
```

Comment:
@mirisuzanne This was the breaking change. The JS was using `pattern-sizes - nav-large` as the breakpoint... now it's `layout-sizes - nav-break`.